pkgdb2/api/extras.py

```diff
@@ -216,7 +218,10 @@ def api_bugzilla():
     acls = _bz_acls_cached(name, out_format)
 
     if out_format == 'json':
-        return flask.jsonify(acls)
+        return flask.Response(
+            str(acls).replace("'", '"').replace('None', 'null'),
```

Comment:
Also, this seems error prone in general.

- Did you try ``import json; json.dumps(acls)``?  Was it faster than ``flask.jsonify(acls)``?
- What if there is a single ``'`` in some string?  Replacing it with ``"`` would be incorrect.
- When converting the string to json, true you need to convert ``None`` to ``null``, but you also need to convert ``True`` to ``true`` and ``False`` to ``false``.